### PR TITLE
Implemented basic layout and Header, SideBar, BottomNavigation components

### PR DESCRIPTION
**Pull request by disha1202**
Screenshot 2023-10-26 at 12 03 24 PM

Screenshot 2023-10-26 at 12 03 08 PM

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -1,11 +1,20 @@
...
+import Header from './components/TheHeader.vue'
+import { isMobile } from './utilities/helper'
 </script>
 
 <template>
-  <RouterView />
+  <v-app>
+    <Header></Header>
+    <SideBar v-if="!isMobile()" />
+    <bottom-navigation v-if="isMobile()" />
```
Comment (AishwaryShrivastav) [inline src/App.vue:13]: This should go below router, for semantics

File: src/components/BottomNavigation.vue
Comment (AishwaryShrivastav) [inline src/components/BottomNavigation.vue:1]: Spacings

Comment (AishwaryShrivastav) [reply src/components/BottomNavigation.vue:1]: order

File: src/components/SideBar.vue [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+            <template v-slot:prepend>
+              <i class="isax isax-setting-2"></i>
+            </template>
+          </v-list-item>
+        </v-list>
+      </div>
+    </template>
+  </v-navigation-drawer>
+</template>
+<style>
```
Comment (disha1202) [inline src/components/SideBar.vue:28]:
```suggestion
<style scoped>
```

File: src/components/SideBar.vue [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+<template>
+  <v-navigation-drawer permanent>
+    <v-list nav>
+      <v-list-item v-for="view in views" :key="view.name" :title="view.name" :value="view.name">
+        <template v-slot:prepend>
+          <i :class="view.icon"></i>
+        </template>
+      </v-list-item>
+    </v-list>
+    <template v-slot:append>
```
Comment (AishwaryShrivastav) [inline src/components/SideBar.vue:14]: add enter

File: src/theme/variable.css [hunk 4]
```diff
@@ -0,0 +1,8 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.isax {
+  font-size: 20px;
```
Comment (AishwaryShrivastav) [inline src/theme/variable.css:7]: rem

File: src/components/TheHeader.vue [hunk 5]
```diff
@@ -0,0 +1,15 @@
+<script setup lang="ts">
+import logo from '../assets/images/logo.svg'
+import { isMobile } from '../utilities/helper'
+</script>
+<template>
+  <v-app-bar>
+    <template v-slot:prepend>
+      <v-img height="30px" width="40px" :src="logo"></v-img>
```
Comment (disha1202) [inline src/components/TheHeader.vue:8]: remove inline css

**Pull request merged**